### DISCUSSION TITLE
[Karmada] Refactor community official website

### DISCUSSION THREAD

**Discussion by nate-double-u** [cncf/mentoring#585] [Google Summer of Code 2022]
[Karmada (Kubernetes Armada)](https://github.com/karmada-io/karmada/) is a Kubernetes management system that enables you to run your cloud-native applications across multiple Kubernetes clusters and clouds. This project is to develop the community official website to hold the necessary documents.

Mentor(s): Hongcai Ren (@RainbowMango)

https://github.com/cncf/mentoring/blob/main/summerofcode/2022.md#refactor-community-official-website

**Comment by rgrupesh** [comment]


**Reply by RainbowMango** [reply]
Link to the original issue: https://github.com/karmada-io/website/issues/14 where documents some requirements.

**Reply by RainbowMango** [reply]
Between March 7 and April 3, Potential GSoC contributors can discuss application ideas with the projects.
